test/functional/test_framework/test_node.py

```diff
@@ -99,17 +100,18 @@ def __init__(self, i, datadir, *, rpchost, timewait, bitcoind, bitcoin_cli, mock
 
     def get_deterministic_priv_key(self):
         """Return a deterministic priv key in base58, that only depends on the node's index"""
+        address_key_pair = collections.namedtuple('address_key_pair', ['address', 'key'])
```

Comment:
namedtuple defines a type: please use CamelCase for the name (e.g. `AddressKeyPair`) and define it outside of the function
(see other uses of namedtuple for consistency)

Reply:
Fixed. Thanks for pointing out, I will make sure to read existing conventions for future PR's.